site/en/docs/extensions/reference/permissions/index.md

```diff
@@ -82,25 +84,25 @@ Most Chrome extension permissions can be specified as optional, with the followi
     </tr>
     <tr id="mdns">
       <td><code>"mdns"</code></td>
-      <td>Grants the extension access to the <a href="mdns">chrome.mdns</a> API.</td>
+      <td>Grants the extension access to the <a href="../mdns">chrome.mdns</a> API.</td>
     </tr>
     <tr id="proxy">
       <td><code>"proxy"</code></td>
-      <td>Grants the extension access to the <a href="mdns">chrome.proxy</a> API to manage Chrome's
-        proxy settings.</td>
+      <td>Grants the extension access to the <a href="../proxy">chrome.proxy</a> API to manage
+        Chrome's proxy settings.</td>
```

Comment:
```suggestion
      <td>Grants the extension access to the
        <a href="/docs/extensions/reference/proxy/">chrome.proxy</a> API to manage Chrome's proxy
        settings.</td>
```

Generally we've prefered using the full absolute path in the extensions docs so that we can use find-replace to update all references in one go.